core/src/apps/benfen/layout.py

```
@@ -0,0 +1,54 @@
+        br_code=ButtonRequestType.SignTx,
+    )
+    return result
+
+
+async def require_confirm_fee(
+    ctx: Context,
+    from_address: str,
+    to_address: str,
+    value: int,
+    gas_price: int,
+    gas_budget: int,
```

Ruff (0.8.2)

Unused function argument: `gas_budget`

(ARG001)